WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -426,12 +426,9 @@ private void UpdateRequiredBtcLabel(ClientRound registrableRound)
 				return; // Otherwise NullReferenceException at shutdown.
 			}
 
-			if (registrableRound == default)
+			if (registrableRound is null && RequiredBTC is null)
```

Comment:
@nopara73 You changed the logic here, was that on purpose?

Reply:
I did not change the logic. Zoom out.

Reply:
You're right: https://github.com/zkSNACKs/WalletWasabi/pull/4360

Reply:
Sorry.